src/components/ui/coaching-sessions/coaching-notes/toolbar.tsx

```diff
@@ -105,7 +117,11 @@ export const Toolbar = () => {
           onClick={() => setIsLinkDialogOpen(true)}
           isActive={editor.isActive("link")}
           icon={<Link className="h-4 w-4" />}
-          title="Link"
+          title={
+            editor.isActive("link")
+              ? "Update link (Ctrl + k)"
+              : "Insert link (Ctrl + k)"
```

Comment:
```suggestion
              ? "Update Link (Ctrl + k)"
              : "Insert Link (Ctrl + k)"
```

This makes it consistent with the other buttons' tooltip capitalization.